dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -575,6 +583,8 @@ def __init__(
         self,
         fargate_scheduler=False,
         fargate_workers=False,
+        fargate_spot=False,
+        fargate_spot_weight=4,
```

Comment:
Again, setting it to a number prevents from getting the value from the config file.